modules/canvaskit/README_BS.md

```diff
@@ -0,0 +1,90 @@
+# Bluescape Extra Notes
+
+# Installation Emscripten
+For emscripten installation use version 2.0.14
+Python 3.0 may not work as described on the website so make it via python 2.7
+So the workable sequence of commands is:
+
+	git clone https://github.com/emscripten-core/emsdk.gi
+	cd ./emsdk
+	python ./emsdk.py install 2.0.14
+	python ./emsdk.py activate 2.0.14
+
+Note the last command!
+echo ... >
+
+# Install skia
+We use our own branch of Skia:
+
+	git remote -v
+		origin  git@github.com:Bluescape/skia.git (fetch)
+		origin  git@github.com:Bluescape/skia.git (push)
+		upstream        https://github.com/google/skia.git (fetch)
+		upstream        https://github.com/google/skia.git (push)
+
+Thus to install:
+	git clone git@github.com:Bluescape/skia.git
+	cd skia
+	python2 tools/git-sync-deps
+
+Another essential is a cmake.
+It might be not present on Mac so:
+	brew install cmake
+
+# Compilation
+Follow the explanations in publish.sh. In brief:
+	make bluescape_release
+or
+	make bluescape_debug
+
+Don't forget to comment out in Makefile. Otherwise it will recompile every time
+from scratch!
+	rm -rf ../../out/canvaskit_wasm
+
+Release take few minutes to compile. Debug takes a while. If failed try to
+simply restart it.
+
+# Connecting to WebC Without Publishing
+Once debugging it might be annoying to publish every time. Make a temporary
+shortcut in:
+	dev/web-frontend/packages/webc/package.json
+	dev/web-frontend/packages/web-main/package.json
+
+Replace:
+ "@bluescape/canvaskit-wasm": ...
+by:
+ "@bluescape/canvaskit-wasm": "link: ../../../skia/modules/canvaskit/canvaskit-debug",
```

Comment:
does this link path still work? Or is it npm_build-debug?

Reply:
There also seems to be a space between link: and the path. 
In general, is this okay? If the package name doesn't match, will the link still work?

Reply:
No , thanks.